src/console_bridgeConfig.cmake

```diff
@@ -1,3 +1,10 @@
 find_package(Rock)
 include(FindPkgConfig)
+
+#add include dir for header
+pkg_search_module(base-logging REQUIRED base-logging)
+include_directories(SYSTEM ${base-logging_INCLUDE_DIRS})
```

Comment:
If this is a public dependency of our console_bridge, it should be exposed as such in the console_bridge .pc file, not here.

Reply:
It is in the pc.in file, but not in this .cmake config for the console_bridge file which is installed (similar to the .pc.in file)

Reply:
https://github.com/rock-core/base-console_bridge/blob/master/src/console_bridge.pc.in#L9

Reply:
This cmake file calls rock_find_pkgconfig which, in turn, does `include_directories` on the resolved packages.

Just noticed the `SYSTEM` part to `include_directories`. This, to me, smells bad. You will need to investigate why the pkg-config resolution does not work.

Reply:
I use currently the following patch on class_loader for the upcoming Debian package release. 
Sorry for the delayed provisioning - I intended to provide all patches once I finalized the release.

```
--- rock-master-20.06-tools-class-loader-0.20171116.orig/src/class_loader.pc.in
+++ rock-master-20.06-tools-class-loader-0.20171116/src/class_loader.pc.in
 Name: @TARGET_NAME@
-Description: @PROJECT_DESCRIPTION@
+Description: "Package for loading plugins at runtime"
 Version: @PROJECT_VERSION@
-Requires: @PKGCONFIG_REQUIRES@
+Requires: console_bridge @PKGCONFIG_REQUIRES@
 Libs: -L${libdir} -l@TARGET_NAME@ @PKGCONFIG_LIBS@
 Cflags: -I${includedir} @PKGCONFIG_CFLAGS@
```

Reply:
How does that resolve the undefined reference to log(), which is added with this patch?